etc/spack/defaults/packages.yaml

```diff
@@ -49,6 +49,7 @@ packages:
       unwind: [libunwind]
       yacc: [bison, byacc]
       flame: [libflame, amdlibflame]
+      uuid: [util-linux-uuid, libuuid]
```

Comment:
It looks like most packages previously depended on `libuuid`. Any reason why you chose `util-linux-uuid` as the default? The fact that it includes `linux` in the name concerns me as a macOS user.

Reply:
Mostly due to https://github.com/spack/spack/issues/10812#issuecomment-590625762, which states that the util-linux version is the only one still maintained. I don't have any strong preference, though.

Reply:
Sounds like a good enough reason to me!